packages/core/src/components/axes/zoom-bar.ts

```diff
@@ -33,6 +33,11 @@ export class ZoomBar extends Component {
 	xScale: any;
 	yScale: any;
 
+	// keep the initial zoomBar domain to avoid the incorrect domain due to data changes
+	defaultZoomBarDomain: any;
```

Comment:
if it's the `initial` zoomBar domain, then shouldn't it be called `initialZoomBarDomain: any;`?

Reply:
because how is `defaultZoomBarDomain` different than what `this.zoomService.getDefaultZoomBarDomain();` will get you?